### PR TITLE
Eliminate vid predication Filter.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Issue(s) number: 
Sub job of #4122 

#### Description:
GetVertices will retrieve Vertex by Id set directly, so we don't filter them again in Filter operator, so we eliminate this Filter operator.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Comment by nevermore3** [comment]
if the vertexIDSeek statement is mixed with other property filtering statements, can we delete the vertexID condition after selecting the vertexID index?

**Comment by Shylock-Hg** [comment]
> if the vertexIDSeek statement is mixed with other property filtering statements, can we delete the vertexID condition after selecting the vertexID index?

We can, but need to refactor VidExtractVisitor and maybe lead to incompatible, so don't refactor it here.

**Review by nevermore3 — APPROVED** [review]

File: src/graph/util/ExpressionUtils.cpp [hunk 1]
```diff
@@ -1205,7 +1205,40 @@ bool ExpressionUtils::checkExprDepth(const Expression *expr) {
...
+  if (expr->kind() == Expression::Kind::kRelIn) {
+    // id(V) IN [List]
+    if (relExpr->right()->kind() != Expression::Kind::kList) {
+      DLOG(ERROR) << "DEBUG POINT:3";
+      return false;
+    }
+  } else if (expr->kind() == Expression::Kind::kRelEQ) {
+    // id(V) = Value
+    if (relExpr->right()->kind() != Expression::Kind::kConstant) {
+      DLOG(ERROR) << "DEBUG POINT:4";
```
Comment (CPWstatic) [inline src/graph/util/ExpressionUtils.cpp:1236]: We don't need these debug log if we had done debug.

**Review by CPWstatic — COMMENTED** [review]
LGTM. Please remove the debug logs.

File: src/graph/util/ExpressionUtils.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/graph/util/ExpressionUtils.cpp]: Fixed.

**Review by nevermore3 — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**